db/dbtypes/types.go

```diff
@@ -484,6 +484,24 @@ func (v VoteChoice) String() string {
 	}
 }
 
+// TicketInfo combines spend and pool statuses and relevant block and spending
+// transaction IDs.
+type TicketInfo struct {
+	Status           string     `json:"status"`
+	PurchaseBlock    *TinyBlock `json:"purchase_block"`
+	MaturityHeight   uint32     `json:"maturity_height"`
+	ExpirationHeight uint32     `json:"expiration_height"`
+	LotteryBlock     *TinyBlock `json:"lottery_block"`
+	Vote             *string    `json:"vote"`
+	Revocation       *string    `json:"revocation"`
```

Comment:
I prefer to have null when these are not valid, rather than omitting or empty strings. But I don't see this done (`*string`) anywhere else, so I'm not sure if this is undesirable or not idiomatic.